test/acceptance/framework/consul/consul_cluster.go

```diff
@@ -459,6 +463,52 @@ func createOrUpdateLicenseSecret(t *testing.T, client kubernetes.Interface, cfg
 	})
 }
 
+// configureSCCs creates RoleBindings that bind the default service account to cluster roles
+// allowing access tot he anyuid and privileged Security Context Constraints on OpenShift.
+func configureSCCs(t *testing.T, client kubernetes.Interface, cfg *config.TestConfig, namespace string) {
+	anyuidClusterRole := "system:openshift:scc:anyuid"
+	privilegedClusterRole := "system:openshift:scc:privileged"
+	anyuidRoleBinding := "anyuid-test"
```

Comment:
This rolebinding is for the consul system components that use the default service account, right? could it make sense to name this `consul-system-anyuid` or `system-anyuid`? I was mixing up this role binding and the ones for static-server/client for a moment while looking at it.

Reply:
This would be for any test pods that aren't setting the service account (currently I think it's just the test-dns pod we use in the consul-dns test). All consul components have their own dedicated service accounts.